### ISSUE TITLE
Making The Forearm And Claw Assembly

### ISSUE THREAD

**Issue by Geckoing** [sta-titansrobotics/27315-IntoTheDeep#11]
Making the Forearm and claw assembly as found on page 12 of the Official 2024-25 REV DUO FTC Starter Bot Build Guide.

[2024-25_REV_DUO_FTC_Starter_Bot-Build_Guide.pdf](https://github.com/user-attachments/files/17348392/2024-25_REV_DUO_FTC_Starter_Bot-Build_Guide.pdf)

**Comment by Geckoing** [comment]
Monday:

I began gathering the parts needed to build the assembly. I was able to gather most of them and began putting together the basic frame needed.

**Comment by Geckoing** [comment]
Tuesday:

I continued to work on the basic frame for the assembly and was able to finish that before continuing on with the instructions. I also helped Ambrose counting out and re-attaching the chain.

**Comment by Geckoing** [comment]
Wednesday:

I added the core hex motor to the side of the frame as well as adding the top cross section with the servo and cable. After that I began page 9 of the instructions.

**Comment by Geckoing** [comment]
Thursday:

I continued to work on the assembly but reached a problem due to the seemingly elusive servo shaft adapters meaning I ended up spending most of my time searching for it. However I was able to get a decent bit done meaning for Friday it is really only the final assembly yet to be done.

**Comment by Geckoing** [comment]
Friday:

Today along with the help of Jadon I was able to gather and complete most of the final steps. However we ran into a problem with the instructions, as on step 14 of the forearm and claw assembly it says to insert the 15mm Spacers in between the 15mm Plastic Rod End Brackets and gear. This simply doesn't work as the 15mm spacers force both sides to far apart and it won't be able to be attached like detailed in the instructions. We were not the only group to encounter this issue as team 27316 also found the same issue.
